src/librustdoc/clean/mod.rs

```diff
@@ -292,7 +292,7 @@ impl Item {
         self.type_() == ItemType::Struct
     }
     pub fn is_enum(&self) -> bool {
-        self.type_() == ItemType::Module
+        self.type_() == ItemType::Enum
```

Comment:
I was *so confused* why enums weren't getting the sidebar stuff, until I decided to trace back through what `is_enum` was doing. Apparently it's been like this for a while, but there's also no other users for this method, so it's not harming anything.